lib/tty/prompt/list.rb

```diff
@@ -111,7 +121,16 @@ def help(value = (not_set = true))
       #
       # @api public
       def default_help
-        self.class::HELP % [enumerate? ? " or number (1-#{@choices.size})" : '']
+        # Note that enumeration and filter are mutually exclusive
+        tokens = if enumerate?
+                   [" or number (1-#{visible_choices.size})", ""]
+                 elsif @choices_filter
+                   ["", ", and alphanumeric/underscore characters to filter"]
```

Comment:
The wording could be a bit more user friendly. Though it is technically correct, I think it is important to remember that CLI users may not necessarily be technically savvy.  I propose something akin to `and press letter keys to filter` or similar.

Reply:
:+1: I've changed `alphanumeric/underscore characters` to `letter keys`, so that it now reads `press Enter to select, and letter keys to filter`.